packages/util/src/errors.ts

```diff
@@ -5,6 +5,18 @@ interface ErrorLike {
   stack: string;
 }
 
+interface HasInnerErrorLike {
```

Comment:
I think that's a little nicer, but feel free to keep the original
```suggestion
interface WithInnerError {
```